draft-ietf-jsonpath-base.md

```diff
@@ -450,7 +450,8 @@ A selector performs its function on each of the nodes in its input
 nodelist, during such a function execution, such a node is referred to
 as the "current node".  Each of these function executions produces a
 nodelist, which are then concatenated into
-the result of the selector.
+the result of the selector. A node may be selected more than once and
```

Comment:
"concatenated into" feels un-idiomatic. "concerted to"?

Reply:
I agree it's un-idiomatic. Not sure about "concerted to". Let me try tweaking the wording.

Reply:
Urgh sorry I meant "concatenated to" or "onto"